### PR TITLE
Assignment-3 Submission

### PR DESCRIPTION
**Pull request by Jai2501**
- Complete Tasks pertaining to Assignment 3

### CODE REVIEW

**Review by Nanxi-Huang — DISMISSED** [review]
great work!👍

File: Frontend/src/Components/LoginSignUp/Login.jsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const Login = ({setAction, setError}) => {
 			</div>
 			<div className="forgot-password">
 				Forgot Password?
-				<span onClick={() => resetUserPasswordUsingFirebase(userEmail)}>
+				<span onClick={() => {resetUserPasswordUsingFirebase(userEmail); setError("Password Reset Email Sent");} }>
 					Click Here.
```
Comment (laihuiqi) [inline Frontend/src/Components/LoginSignUp/Login.jsx:71]: Good modification to add convenience to users!

**Review by laihuiqi — DISMISSED** [review]
LGTM as for Asg 3 :+1:

**Review by Nanxi-Huang — DISMISSED** [review]
LGTM👍

**Review by laihuiqi — DISMISSED** [review]
LGTM!

**Review by laihuiqi — DISMISSED** [review]
LGTM!

**Review by laihuiqi — DISMISSED** [review]
LGTM

**Review by Jai2501 — COMMENTED** [review]
LGTM!

**Review by laihuiqi — DISMISSED** [review]
LGTM!

**Review by Linda124 — APPROVED** [review]
lgtm

**Review by Linda124 — DISMISSED** [review]
lgtm

**Pull request merged**